packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -1334,6 +1340,7 @@ describe('plugin-meetings', () => {
               },
             },
           });
+          Media
```

Comment:
what's this about? :-)

Reply:
removed. I am surprised that this didn't cause a test failure.